src/java.base/share/classes/java/lang/Runtime.java

```diff
@@ -655,6 +655,10 @@ public Process exec(String[] cmdarray, String[] envp, File dir)
      * There is no guarantee that this effort will recycle any particular
      * number of unused objects, reclaim any particular amount of space, or
      * complete at any particular time, if at all, before the method returns or ever.
+     * There is also no guarantee that this effort will determine
+     * the change of reachability in any particular number of objects
+     * or any particular number of {@link java.lang.ref.Reference Reference}
+     * objects be cleared and enqueued.
```

Comment:
perhaps
s/number of objects or any/number of objects, or that any/
s/objects be/objects will be/

Reply:
That may be clearer.  Updated.